src/main.cpp

```diff
@@ -818,15 +821,20 @@ bool AreInputsStandard(const CTransactionBase& txBase, const CCoinsViewCache& ma
     if (txBase.IsCoinBase())
         return true; // Coinbases don't use vin normally
 
-    for(const CTxIn& in : txBase.GetVin()) {
-        const CTxOut& prev = mapInputs.GetOutputFor(in);
+    for (unsigned int i = 0; i < txBase.GetVin().size(); i++)
```

Comment:
Consider using:
 const CTxin& in = txBase.GetVin().at(i)
for better reading and also since it is used twice in this block

Reply:
Done